UNRELEASED.yaml

```diff
@@ -11,6 +11,11 @@ FIXED:
   - bpk-component-autosuggest:
     - Upgraded `react-autosuggest` to `10.0.2`.
 
+BREAKING/ADDED:
```

Comment:
Wasn't sure if this was Breaking or just Added due to me removing code. Wdyt?

Reply:
The removed code is all in non-public files, right? So non-breaking

Reply:
Unless we consider the visual changes to be breaking?

Reply:
Made it just added